protocol/test/core/vault-proxy-tests.ts

```diff
@@ -231,7 +237,8 @@ describe("Vault Proxy", async () => {
   })
 
   it("Can proxy deposit for any Vault", async() => {
-    await TEMPLE.connect(alan).increaseAllowance(VAULT_PROXY.address, toAtto(1000));
+    await TEMPLE.connect(alan).approve(VAULT_PROXY.address, 0);
```

Comment:
I'm surprised we need to approve(0) first for the Temple ERC20. I thought it was only USDT that needed it as it's custom.

Has something changed in ERC20 where this is required now

Reply:
after checking the ERC20 contract inherited, it's fine to use one approve for temple token